tests/system/provisioning/roles/aws/templates/ansible_hosts.j2

```diff
@@ -318,17 +343,8 @@ windowsagents:
         ansible_connection: winrm
         ansible_winrm_transport: basic
         ansible_winrm_server_cert_validation: ignore
-        wazuh_winagent_config:
-            version: {{ windows_agent_version }}
-{% if packages_repository != 'production' %}
-{% if packages_repository == 'staging' %}
-            repo: 'https://packages-dev.wazuh.com/pre-release/windows/'
-            md5: {{ windows_md5 }}
-{% elif packages_repository == 'devel' %}
-            repo: 'https://packages-dev.wazuh.com/staging/windows/'
-            md5: {{ windows_md5 }}
-{% endif %}
-{% endif %}
+        wazuh_winagent_config_url: https://s3-us-wes-1.amazonaws.com/packages-dev.wazuh.com/warehouse/branches/3.12/windows/wazuh-agent-3.12.0-0.3319fimreworksqlite.msi
+        wazuh_winagent_package_name: wazuh-agent-3.12.0-0.3319fimreworksqlite.msi
```

Comment:
Why don't use `wazuh_winagent_config_url` and `wazuh_winagent_package_name` variables?

Reply:
Done in 872f0b9b52219ef51a929b640f7f51eabf5fcfe4